pages/plot-data-from-csv.py

```diff
@@ -3,21 +3,22 @@
 from utils.code_and_show import example_app
 
 
-dash.register_page(__name__, description="Plotting data from csv file")
+dash.register_page(__name__, description="This application plots stocks data collected from csv. It contains button to invert axis on graph.")
 
 filename = __name__.split("pages.")[1]
 
 
 notes = """
+#### Dash Components in App:
+- [Button](https://dash.plotly.com/dash-html-components/button)
+- [Graph](https://dash.plotly.com/dash-core-components/graph)
+- [Line](https://plotly.com/python/line-charts/)
```

Comment:
Please move this to below the Plotly Documentation section because this link comes from the Plotly Graphs.